trex/mini2Dx-sample-trex-runner/core/build/tmp/jar/MANIFEST.MF

```diff
@@ -0,0 +1,2 @@
+Manifest-Version: 1.0

```

Comment:
You'll also need to delete the build folder, make a commit, then squash the commit to remove the files for Github. It's generally best practice not to commit build files into the repo so that other developers don't get your files.